src/streamlink_cli/argparser.py

```diff
@@ -213,6 +213,13 @@ def keyvalue(value):
     """
 )
 general.add_argument(
+    "--can-handle-url-nohead",
+    metavar="URL",
+    help="""
+    Same as --can-handle-url but without HTTP operations involved.
```

Comment:
Maybe `Same as --can-handle-url but without following redirects when looking up the URL` is more clear about the indented function?